packages/plugins/ui-theme/src/App/Header/Header.test.tsx

```diff
@@ -34,6 +35,7 @@ describe('<Header /> component with logged in state', () => {
     expect(container.firstChild).toMatchSnapshot();
     expect(queryByTestId('header--menu-accountcircle')).toBeNull();
     expect(getByText('Login')).toBeTruthy();
+    expect(screen.queryByTestId('header--button-login')).toBeInTheDocument();
```

Comment:
Hm, do you need that screen. ?

Reply:
yep, it is the recommended usage https://testing-library.com/docs/queries/about/#screen

When these test were created there was no `screen`, the code should move to only rely on `screen`.

Reply:
But I don't understand this, because in the line 36, you're using queryByTestId without that screen. access. 

I thought when i've used testing library that screen is required when you don't want to use the const destructuring of the render method, in this case you're already doing it.

Reply:
> But I don't understand this, because in the line 36, you're using queryByTestId without that screen. access.

That's leftover, I just clean it up. More info here https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
I also removed the snapshots, https://github.com/verdaccio/ui/issues/199 that is a pending task from quite while ago.

Reply:
Ah cool, didn't know that :)